plugins/manager/lib/yform/manager.php

```diff
@@ -1552,4 +1552,18 @@ public function createTable($mifix, $data_table, $params = [], $debug = false)
 
         return true;
     }
+
+    public static function url(string $tableName, int $id, array $params = []): string
```

Comment:
Braucht man auch die Möglichkeit, die URL zu einer Tabelle zu liefern, also nicht in den Datensatz hinein? Dann könnte man `$id` optional machen.

Wobei ich mich gleichzeitig gerade frage, ob es vielleicht doch schöner wäre, direkt den Datensatz übergeben zu können, also `rex_yform_manager_dataset $dataset` als einen Parameter (statt der zwei für table und id). Das wäre wiederum inkompatibel zu meiner Idee oben. 

Man könnte es dann auch doch aufteilen auf zwei Methode, `tableUrl` und `datasetUrl` (oder `urlFor...` oder so).

Reply:
Oder vielleicht sogar `$dataset->getUrl()`? Wobei ich nicht weiß, ob ich das so wirklich integrieren würde. Ggf. möchte man den Methodennamen eher für was eigenes in seiner Modelklasse verwenden. Und die Datasets sind ja eigentlich mehr Frontend-Zeugs, während es hier um die Backend-URL geht.

Reply:
Als Hintergrund … Watson benötigt eine Url zum Datensatz.
https://github.com/tbaddade/redaxo_watson/pull/63/files#diff-98f48dec91101cd112c53cc12a5d62885a1bbc77f12ca5e221aef3597b66b830R115-R117

Hier erst den Datensatz zu holen, weiß nicht so recht.

> Oder vielleicht sogar $dataset->getUrl()? Wobei ich nicht weiß, ob ich das so wirklich integrieren würde. Ggf. möchte man den Methodennamen eher für was eigenes in seiner Modelklasse verwenden.

Das passiert durchaus bereits in Kombination mit dem Url-AddOn

```php
public function getUrl()
{
    return rex_getUrl('', '', ['model-id' => $this->getId()]);
}
```

Reply:
> Hier erst den Datensatz zu holen, weiß nicht so recht.

Ok hast recht. Dann wäre mein Vorschlag, die Methode vom Prinzip so zu lassen, wie du es jetzt hier im PR hast. Bloß den `$id`-Parameter würde ich wahrscheinlich optional machen.